src/pages/SignupPage/SignUpPage.jsx

```diff
@@ -1,10 +1,12 @@
 import React from "react";
 import { Navbar, Footer, Input, Label } from "../../components";
 import { Link } from "react-router-dom";
+import { useDocumentTitle } from "../../hooks";
 import "./sign-up-page.css";
 import { BiEye, BiEyeSlash } from "../../assets/icons";
 
 export function SignUpPage() {
+  useDocumentTitle("Signup");
   return (
     <div classNameName='signup-container'>
       <Navbar />
```

Comment:
Generally signup page should not have Navbar

Reply:
Thanks for reviewing. I have kept it till the time I add authentication.